src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -0,0 +1,208 @@
+
++   Routing with default code-splitting
++   Compilation and bundling (using [Babel](https://babeljs.io/) and
+    [webpack](https://webpack.js.org/))
++   Server-side rendering
++   Asynchronous data fetching for every page
++   Default data store ([Vuex](https://vuex.vuejs.org/guide/))
+
+Along with working directly on improving the performance of different tools, we've expanded the
+framework fund to provide monetary support to more open-source frameworks and libraries. With our
+[recent support](https://github.com/nuxt/nuxt.js/issues/6467#issuecomment-538192059) to Nuxt.js, a
+few features are slated to land in the near future including smarter server-rendering and image
+optimizations.
+
+{% Aside %}
+The [framework fund](https://opencollective.com/chrome) accelerates the efforts of different
+frameworks and libraries with the goal to improve their performance. If you are working on an
+open-source tool and need support, [please apply](http://bit.ly/chrome-framework-fund)!
+{% endAside %}
+
+## Babel
+
+We've also made progress on improving the performance of an important underlying tool in almost all
+of the mentioned frameworks--[Babel](https://babeljs.io/).
+
+Babel compiles code that contains newer syntax into code that different browsers can understand.
+It's become common to use [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) to target
+modern browsers where different browser targets can be specified to provide enough polyfilling
+required for all the chosen environments. One way to specify the targets is to use `<script
+type="module">` to target all browsers that [support ES
+Modules](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules).
+
+{% Aside %}
+To learn more about how to leverage ES modules to minimize polyfill code shipped to browsers
+that do not need them check out
+["Serve modern code to modern browsers for faster page loads"](../serve-modern-code-to-modern-browsers).
+{% endAside %}
+
+To optimize for this case, we launched a brand new preset;
+[@babel/preset-modules](http://github.com/babel/preset-modules). Instead of converting modern syntax
+to older syntax to avoid browser bugs, preset-modules fixes each specific bug by transforming to the
+closest possible non-broken modern syntax. This results in modern code that can be delivered_ nearly
+unmodified _to most browsers.
+
+![A new babel preset to provide better polyfilling for browsers](./babel-preset-env.png)
+
+Developers who already use preset-env will also benefit from these optimizations without having to
+do anything, as they'll soon be incorporated into preset-env too.
+
+## What's next?
+
+Working closely with open-source frameworks and libraries to provide better experiences helps the
+Chrome team realize what is fundamentally important to users and developers alike.
+
+If you work on a web framework, UI library, or any form of web tooling (bundler, compiler, linter),
+[reach out to us](https://twitter.com/ChromiumDev)!
```

Comment:
```suggestion
[apply for the framework fund](http://bit.ly/chrome-framework-fund)!
```
Maybe a more powerful call-to-action? Feel free to ignore.

Reply:
Agreed 👍